### ISSUE TITLE
Select: <svg> elements with an img role must have an alternative text

### ISSUE THREAD

**Issue by techie-khushdeep** [SAP/ui5-webcomponents-react#7014]
### Describe the bug

Hi,
We are using select component in our application. We are getting a11y issue in axe dev tools as well as jest test.
**A11y issue**: Ensure <svg> elements with an img, graphics-document or graphics-symbol role have an accessible text.

Issue screenshot:

![Image](https://github.com/user-attachments/assets/e3609a56-39d4-4d9d-91ff-4368d9e1e2b0)

**official component url**: https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/inputs-select--docs

### Isolated Example

https://stackblitz.com/edit/github-h2thrpbd-dhjhe67o?file=src%2FApp.tsx

### Reproduction steps

1. Run the application using https://stackblitz.com/edit/github-h2thrpbd-dhjhe67o?file=src%2FApp.tsx
2. Do the scan using axe browser extension and check the a11y issue.

### Expected Behaviour

There should be no a11y issue for component.

### UI5 Web Components for React Version

2.5.0

### UI5 Web Components Version

2.5.0

### Browser

Chrome

### Relevant log output

```shell

```

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
